pkg/resources/spark/configmap_test.go

```diff
@@ -0,0 +1,170 @@
+package spark
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/dominodatalab/distributed-compute-operator/api/v1alpha1"
+)
+
+func TestNewFrameworkConfigMap(t *testing.T) {
+	t.Run("fully loaded", func(t *testing.T) {
+		rc := sparkClusterFixture()
+		rc.Spec.Master.FrameworkConfig = &v1alpha1.FrameworkConfig{
+			Configs: map[string]string{
+				"m1": "v1",
+				"m2": "v2",
+			},
+		}
+		rc.Spec.Worker.FrameworkConfig = &v1alpha1.FrameworkConfig{
+			Configs: map[string]string{
+				"w1": "v1",
+				"w2": "v2",
+			},
+		}
+		cm := NewFrameworkConfigMap(rc)
+
+		expected := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "test-id-framework",
+				Namespace: "fake-ns",
+				Labels: map[string]string{
+					"app.kubernetes.io/name":       "spark",
+					"app.kubernetes.io/instance":   "test-id",
+					"app.kubernetes.io/version":    "fake-tag",
+					"app.kubernetes.io/managed-by": "distributed-compute-operator",
+				},
+			},
+			Data: map[string]string{
+				"master": `m1 v1
+m2 v2
+`,
```

Comment:
```suggestion
				"master": "m1 v1\nm2 v2",
```

Is this not a cleaner way of writing this string instead of using raw string literal?